src/main/java/com/epam/ta/reportportal/core/analyzer/auto/client/impl/AnalyzerServiceClientImpl.java

```diff
@@ -76,6 +80,41 @@ public boolean hasClients() {
 		return search(rq, analyzerExchanges);
 	}
 
+	@Override
+	public List<SuggestInfo> searchSuggests(SuggestRq rq) {
+		final List<ExchangeInfo> exchangeInfos = rabbitMqManagementClient.getAnalyzerExchangesInfo()
```

Comment:
Can be little reduced. Something like that:
`		final String exchangeName = rabbitMqManagementClient.getAnalyzerExchangesInfo()
				.stream()
				.filter(DOES_SUPPORT_SUGGEST)
				.min(Comparator.comparingInt(EXCHANGE_PRIORITY))
				.map(ExchangeInfo::getName)
				.orElseThrow(() -> new ReportPortalException(ErrorType.UNABLE_INTERACT_WITH_INTEGRATION,
						"There are no analyzer services with suggest items support deployed."
				));

		return rabbitTemplate.convertSendAndReceiveAsType(exchangeName,
				SUGGEST_ROUTE,
				rq,
				new ParameterizedTypeReference<>() {
				}
		);

`